src/main/java/net/dv8tion/jda/internal/utils/PermissionUtil.java

```diff
@@ -356,7 +356,7 @@ public static long getEffectivePermission(IPermissionContainer channel, Member m
         if (isApplied(permission, admin))
             return Permission.ALL_PERMISSIONS;
         if (member.isTimedOut())
-            return Permission.getRaw(Permission.VIEW_CHANNEL, Permission.MESSAGE_HISTORY);
+            return permission & Permission.getRaw(Permission.VIEW_CHANNEL, Permission.MESSAGE_HISTORY);
```

Comment:
This should go at the bottom, after the permissions of overwrites were applied. Currently, you just forward the role permissions without taking channel permissions into account.

Reply:
I thought it didn't matter because the member would have no permissions anyway, but didn't think of the possibility of all channel overrides being disabled